web/service/src/main/java/org/mobilitydata/gtfsvalidator/web/service/controller/ValidationController.java

```diff
@@ -43,6 +45,9 @@ public class ValidationController {
   @Autowired private StorageHelper storageHelper;
   @Autowired private ValidationHandler validationHandler;
 
+  @Autowired private VersionResolver versionResolver;
+  private static final Duration TIMEOUT = Duration.ofSeconds(5);
```

Comment:
Please remove the TIMEOUT field, as it is not being used anymore.

Reply:
removed